src/Plug/Widget/Widget.h

```diff
@@ -0,0 +1,130 @@
+#ifndef WIDGET_H
+#define WIDGET_H
+
+#include "Plug/Graphics.h"
+
+class LayoutBox;
+
+class EHC;
+
+class Event;
+class TickEvent;
+class MouseMoveEvent;
+class MousePressedEvent;
+class MouseReleasedEvent;
+class KeyboardPressedEvent;
+class KeyboardReleasedEvent;
+
+class Vec2d;
+class TransformStack;
```

Comment:
Don't add definitions of this classes here. They will be added in other branches later.